#19 replica fix (#15)
manticoresoftware/manticoresearch-helm: README.MD

  b) of it's not set - from the host
5) And run OPTIMIZE with `cutoff=<cpuCores * values.optimize.coefficient>`

> ### How replication work?
> Replication script affects only worker pods. In each start executes schema below:
> ![Replication workflow](replication.jpg)

Reviewer: ```suggestion
### How does replication work?
The replication script affects only worker pods. In a nutshell it works like this:
![Replication workflow](replication.jpg)
```

Reviewer: This is not resolved https://github.com/manticoresoftware/manticoresearch-helm#how-replication-work

<img width="745" alt="image" src="https://user-images.githubusercontent.com/28719845/174278601-f7f06e21-7107-4d68-90df-411524874317.png">
